articles/tech-resources/change-log-sparkpost-enterprise.md

```diff
@@ -6,6 +6,23 @@ notification: "This Change Log is for SparkPost Enterprise Only"
 
 A running log of new features and capabilities that have been rolled out on the SparkPost Enterprise service. The API reference can be found [here](https://developers.sparkpost.com/api).
 
+### (Week of) 11 September, 2017 
+
+* **Change:** Message Scheduling
+  *  We have changed the behavior of the scheduled send function. Prior to the change schedule time kicked off message *generation* and then the immediate send of the generated messages. With this change, message injected into SparkPost will begin generating immediately and held in queue. The scheduled time will being message *sending.*
+  * This change will result the actual sending time to be much closer to the scheduled time because of the pre-generation.
```

Comment:
`...will bring actual sending time much closer to the scheduled time because scheduled mail is pre-generated.`?